patches/chrome-browser-ui-views-tabs-tab.cc.patch

```diff
@@ -10,7 +10,15 @@ index 7b4670597248635874b2b1650b1eae40fddb52e7..1888150f0ad27c60b19bc451c6a70eb7
      } else if (showing_close_button_) {
        // Allow the title to overlay the close button's empty border padding.
        title_right = close_x - after_title_padding;
-@@ -1009,6 +1010,7 @@ void Tab::UpdateIconVisibility() {
+@@ -643,6 +644,7 @@ void Tab::OnGestureEvent(ui::GestureEvent* event) {
+ }
+ 
+ std::u16string Tab::GetTooltipText(const gfx::Point& p) const {
++  BRAVE_TAB_HOVER_TOOLTIPS
```

Comment:
this doesn't seem like the right place to make this change per the comment below

Reply:
I guess we're providing a replacement to hover cards, but where does it handle hover cards now?

Reply:
also this could be accomplished by subclassing Tab since this a virtual method